### PR TITLE
Max to maximum

### PR DESCRIPTION
**Pull request by toikarin-vincit**
7069dc3: frontend: fix linting error that prevented the build
 - `Definition for rule 'react-hooks/exhaustive-deps' was not found  react-hooks/exhaustive-deps`

---

4d5c9fa: backend/frontend: change `max_price(s)` -> `maximum_prices` in JSON responses

---

44e8e02: backend: Change `MaxPrice` -> `MaximumPrice` in codebase and models

---

3b651be: backend: add possibility to check if oracle migration has been executed
 - previously starting the service with both oracle migration done and
   `LOAD_INITIAL_DATASET=1` failed due to key errors (same usernames
   or postal codes were imported from both places)
 - now `./manage hitasmigrate` will check if migration has been done
   previously before starting and refuse to start if it has already
   been completed (unless `--truncate` is given)
 - `./manage hitasmigrate` has a new option `--check` which only
   checks if migration has done previously
 - `docker-entrypoint.sh` will check if oracle migration has been
   completed before loading initial dataset and ignore it if it
   has

---

6f43166: docker-compose: add oracle image

---

fc74fae: backend: update dependencies

---

85b9b40: frontend: update dependencies

---

c3f04c5: docker-compose: update swaggerapi/swagger-editor

### CODE REVIEW

**Comment by ranta** [comment]
Can you discard (or update) the frontend modifications, please?

File: backend/hitas/models/apartment.py [hunk 1]
```diff
@@ -158,7 +158,7 @@ class ApartmentConstructionPriceImprovement(HitasImprovement):
     depreciation_percentage = EnumField(DepreciationPercentage, default=DepreciationPercentage.TEN)
 
 
-class ApartmentMaxPriceCalculation(models.Model):
+class ApartmentMaximumPriceCalculation(models.Model):
```
Comment (ranta) [inline backend/hitas/models/apartment.py:161]: Was this model's `max_price = models.IntegerField(...)` intentionally not renamed? 🤔

Comment (toikarin-vincit) [reply backend/hitas/models/apartment.py:161]: not really, fixed

**Comment by toikarin-vincit** [comment]
> Can you discard (or update) the frontend modifications, please?

updated

**Review by ranta — APPROVED** [review]
LGTM 🚀

**Pull request merged**